src/fastmcp/utilities/skills.py

```
@@ -0,0 +1,250 @@
+            print(f"Downloaded to: {skill_path}")
+        ```
+    """
+    target_dir = Path(target_dir).expanduser().resolve()
+    skill_dir = target_dir / skill_name
+
+    # Check if directory exists
+    if skill_dir.exists() and not overwrite:
+        raise FileExistsError(
+            f"Skill directory already exists: {skill_dir}. "
+            "Use overwrite=True to replace."
+        )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)